### PR TITLE
add tags and labels for aws and gcp instances

### PR DESCRIPTION
**Pull request by arturrez**
resolves https://github.com/ava-labs/avalanche-cli/issues/959

### CODE REVIEW

**Review by felipemadero — COMMENTED** [review]
nice add on for when using aws ux. I created another one for static/elastic ips maybe can be addressed here https://github.com/ava-labs/avalanche-cli/issues/1385

File: pkg/cloud/aws/aws.go [hunk 1]
```diff
@@ -146,6 +146,21 @@ func (c *AwsCloud) CreateEC2Instances(count int, amiID, instanceType, keyName, s
 				},
 			},
 		},
+		TagSpecifications: []types.TagSpecification{
```
Comment (felipemadero) [inline pkg/cloud/aws/aws.go:149]: can we do the same with static ip names?

File: pkg/cloud/gcp/gcp.go [hunk 2]
```diff
@@ -262,6 +262,10 @@ func (c *GcpCloud) SetupInstances(zone, networkName, sshPublicKey, ami string, s
 				Scheduling: &compute.Scheduling{
 					AutomaticRestart: &automaticRestart,
 				},
+				Labels: map[string]string{
```
Comment (felipemadero) [inline pkg/cloud/gcp/gcp.go:265]: can we do the same with static ip names?

Comment (arturrez) [reply pkg/cloud/gcp/gcp.go:265]: sure

File: pkg/cloud/gcp/gcp.go [hunk 3]
```diff
@@ -262,6 +262,10 @@ func (c *GcpCloud) SetupInstances(zone, networkName, sshPublicKey, ami string, s
...
+					"name":       "avalanche",
```
[the remark this answers is not in the document]
Comment (arturrez) [reply pkg/cloud/gcp/gcp.go:266]: sure

File: pkg/cloud/aws/aws.go (hunk 1 above)
Comment (arturrez) [reply pkg/cloud/aws/aws.go:149]: sure

**Comment by arturrez** [comment]
image

File: pkg/cloud/aws/aws.go [hunk 4]
```diff
@@ -146,6 +146,21 @@ func (c *AwsCloud) CreateEC2Instances(count int, amiID, instanceType, keyName, s
...
+			{
+				ResourceType: "instance",
+				Tags: []types.Tag{
+					{
+						Key:   aws.String("Name"),
+						Value: aws.String("avalanche"),
```
[the remark this answers is not in the document]
Comment (arturrez) [reply pkg/cloud/aws/aws.go]: fixed

**Review by felipemadero — APPROVED** [review]

**Pull request merged**